man/fi_shm.7.md

```diff
@@ -25,7 +25,7 @@ offers the following support:
 : The provider supports only endpoint type *FI_EP_RDM*.
 
 *Endpoint capabilities*
-: Endpoints cna support any combinations of the following data transfer
+: Endpoints can support any combinations of the following data transfer
```

Comment:
Please move the shm changes to a separate commit and/or PR

Reply:
Ah, I see these are for another provider. I will remove them from this PR and handle outside of this change.